playground/mocks/data/idp/idx/authenticator-verification-okta-verify-select-method.json

```diff
@@ -34,15 +34,15 @@
                   "required": true,
                   "options": [
                     {
-                      "label": "Use Okta FastPass",
+                      "label": "FastPass",
```

Comment:
Deliberately made this to a different string so tests can check if we transform correctly.